flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -126,7 +126,7 @@ private Configuration fromPackagedProgram(final PackagedProgram program, final i
 	 */
 	@Test
 	public void testDetachedMode() throws Exception{
-		final ClusterClient<?> clusterClient = new MiniClusterClient(new Configuration(), MINI_CLUSTER_RESOURCE.getMiniCluster());
+		final ClusterClient<MiniClusterClient.MiniClusterId> clusterClient = new MiniClusterClient(new Configuration(), MINI_CLUSTER_RESOURCE.getMiniCluster());
```

Comment:
Does adding the type here instead of `?` solve anything? It seems it just makes the code less readable.

Reply:
Nope, it is an intermediate change. Will revert.